py3-sphinxext-opengraph.yaml

```diff
@@ -0,0 +1,77 @@
+package:
+  name: py3-sphinxext-opengraph
+  version: 0.10.0
+  epoch: 0
+  description: sphinxcontrib-devhelp is a sphinx extension to generate unique OpenGraph metadata
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provider-priority: "0"
+
+vars:
+  pypi-package: sphinxext-opengraph
+  import: sphinxext.opengraph
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '313'
```

Comment:
no need for the values here, just leave them empty.

Reply:
done.